libs/utils/env.py

```diff
@@ -239,17 +248,26 @@ def __init__(self, target_conf=None, test_conf=None, wipe=True,
         if '__features__' not in self.conf:
             self.conf['__features__'] = []
 
-        # Initialize local results folder
-        # test configuration overrides target one
-        self.res_dir = (self.test_conf.get('results_dir') or
-                        self.conf.get('results_dir'))
-
-        if self.res_dir and not os.path.isabs(self.res_dir):
-            self.res_dir = os.path.join(basepath, 'results', self.res_dir)
-        else:
+        # Initialize local results folder.
+        # The test configuration overrides the target's one and the environment
+        # variable overrides everything else.
+        self.res_dir = (
+            os.getenv('LISA_RESULTS_DIR') or
+            self.conf.get('results_dir') or
+            # THIS MUST NOT BE USED. It is only available for a specific test,
```

Comment:
Since we have control on all tests... I'm more of the idea to have a single API, which is an improvement of the current one, and update the tests to use the new one.

Reply:
Ok, I'll update these tests so they don't rely on this key in test conf.